src/components/appSwitch/AppSwitch.js

```diff
@@ -181,6 +184,7 @@ const AppSwitch = (props: LoadingProps) => {
           gdstore.set('isLoggedInCitizen')(false)
         }
       }
+      identifyWith(userStorage.getProfileFieldValue('email'), goodWallet.getAccountForType('login'))
```

Comment:
```suggestion
      const [email, login] = await Promise.all(['email', 'login'].map(idField => goodWallet.getAccountForType(idField)))

      identifyWith(email, login)
```

Reply:
shouldn't it be await ? userStorage.getProfileFieldValue is async method, as i remember

Reply:
yes you are right